lib/ClangImporter/ImportDecl.cpp

```diff
@@ -5393,7 +5393,7 @@ Decl *SwiftDeclConverter::importCompatibilityTypeAlias(
       alias->setGenericParams(GTD->getGenericParams()->clone(alias));
   }
 
-  alias->setUnderlyingType(Impl.getSugaredTypeReference(typeDecl));
+  alias->setUnderlyingType(typeDecl->getDeclaredInterfaceType());
```

Comment:
This looks like part of a separate patch, but it also looks like a good change to me.

Reply:
Probably, but it's such a small change it would inflate my commit / lines deleted ratio. :) In case it's not apparent to someone reading the patch, the reason getSugaredTypeReference() is no longer needed is that at some point getDeclaredInterfaceType() was changed from returning the underlying type to returning the sugared type for type aliases. It wasn't always the case.